src/main/java/org/takes/rq/form/package-info.java

```diff
@@ -32,11 +32,20 @@
  *
  * <p>All implementations of this interface must be immutable and thread-safe.
  *
+ * Current implementations:
+ * {@link org.takes.rq.form.RqFormBase} base implementation;
+ * {@link org.takes.rq.form.RqFormFake} implementation that useful for
+ * testing purposes;
+ * {@link org.takes.rq.form.RqFormSmart} decorator with extra features.
+ *
+ * Also please look at usage examples:
+ *
+ * {@link org.takes.rq.form.RqFormBaseTest};
```

Comment:
@xupyprmv I'm not sure here... for someone using takes as dependency, these links won't be working